8.0 release breaks command line checks for ES5 ("Cannot find module './detectFeatures'")

Steps to reproduce, using a fresh environment (no preinstalled packages):

- `npm install es-check`
- `npx es-check es5`

Resulting log:

```
~/test $ npx es-check es5
node:internal/modules/cjs/loader:1148
  throw err;
  ^

Error: Cannot find module './detectFeatures'
Require stack:
- /home/zeux/test/node_modules/es-check/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/home/zeux/test/node_modules/es-check/index.js:12:24)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/zeux/test/node_modules/es-check/index.js' ]
}

Node.js v20.16.0
```

--- a/package.json
+++ b/package.json
@@ -5,7 +5,10 @@
   "main": "index.js",
   "license": "MIT",
   "files": [
-    "index.js"
+    "index.js",
+    "utils.js",
+    "detectFeatures.js",
+    "constants.js"
   ],
   "bin": {
     "es-check": "index.js"

